dojo/templates/dojo/snippets/tags.html

```diff
@@ -0,0 +1,13 @@
+{% if tags %}
```

Comment:
Good idea to implement a snippet. Is that covered in our integration tests? There are lot of changes in the templates

Reply:
Have any suggestions? The content of displaying the tags is not changing.